<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>327</width>
    <height>135</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Number Converter</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="labelHex">
     <property name="text">
      <string>Hex</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="labelOct">
     <property name="text">
      <string>Oct</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelDec">
     <property name="text">
      <string>Dec</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="labelBin">
     <property name="text">
      <string>Bin</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineEditHex"/>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="lineEditDec"/>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="lineEditOct"/>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="lineEditBin"/>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="labelQuickStartGuide">
     <property name="text">
      <string>Input Number, then press Enter</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
